Sources/Purchasing/Purchases/PurchasesOrchestrator.swift

```diff
@@ -1944,6 +1938,56 @@ private extension PurchasesOrchestrator {
 
 }
 
+// MARK: - Test Store Purchases
+
+private extension PurchasesOrchestrator {
+
+    func handlePurchase(testStoreProduct: TestStoreProduct, completion: @escaping PurchaseCompletedBlock) {
```

Comment:
I see we created a `TestStorePurchaseHandler`. Would it make sense to also move all this logic there? Mostly want to avoid increasing the size of the orchestrator too much... and this all seems like it could live there?

Reply:
Yes, I think that's a good point. I intend to add the whole after-purchase logic in a separate PR so I'll take this into account to add all the logic in the `TestStorePurchaseHandler`. I agree it's better to keep `PurchasesOrchestrator` as dedicated as possible to the "real" purchases logic.
Thanks for the suggestion!